.buildkite/data.rayci.yml

```diff
@@ -55,7 +55,7 @@ steps:
       - python
       - data
     # run only on postmerge
-    # if: build.env("BUILDKITE_PIPELINE_ID") == "0189e759-8c96-4302-b6b5-b4274406bf89"
+    if: build.env("BUILDKITE_PIPELINE_ID") == "0189e759-8c96-4302-b6b5-b4274406bf89"
```

Comment:
please remove this line, and add `- skip-on-premerge` in the `tags` part.

Reply:
Fixed